supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -7,3 +7,26 @@
 
 [[external-links]]
 == External links
+
+[[rh-kb-links]]
+== Linking to Red Hat Knowledgebase articles
+
+*Guidance for Red Hat documentation*
+
+Guidelines:
```

Comment:
Suggest using an intro sentence instead here. Something like "Follow these guidelines when linking to a Red Hat Knowledgebase article or solution:".

Reply:
@bergerhoffer  That seems like fluff to me. I looked at other topics in the style guide and they just start with the guideline immediately following the heading. For example *Graphical interfaces* and *General formatting*. So I just removed this line.

Reply:
That works!